docs/infra/infrastructure-configuration.md

```diff
@@ -0,0 +1,30 @@
+# Infrastructure configuration
+
+## Configure infrastructure with static configuration modules
+
+The infrastructure derives all of its configuration from static configuration modules:
```

Comment:
Might be worth defining what "static" means here.

Reply:
Added a section